NuRadioReco/modules/io/coreas/readCoREASStation.py

```diff
@@ -21,14 +24,20 @@ def begin(self, input_files, station_id):
         """
         self.__input_files = input_files
         self.__station_id = station_id
-        self.__current_input_file = 0        
+        self.__current_input_file = 0
 
     def run(self, detector):
         for input_file in self.__input_files:
 
             # read in coreas simulation and copy to event
+#             if input_file == "../../data/CR_sims/005548.hdf5":
```

Comment:
Why do we need this line? We should not hard-code this anyway.

Reply:
Was commented out already anyway. Removed it for cosmetic reasons now in any case.